### PR TITLE
update Kubernetes node presubmits

### PR DESCRIPTION
**Pull request by BenTheElder**
- switch to quick build
- drop stray presets

/cc @dims @yujuhong

### CODE REVIEW

**Comment by BenTheElder** [comment]
broken out of https://github.com/kubernetes/test-infra/pull/21039

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -62,8 +62,6 @@ presubmits:
     max_concurrency: 12
     labels:
       preset-service-account: "true"
-      preset-bazel-scratch-dir: "true"
```
Comment (dims) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:65]: don't need dind preset here? (like the other two)

Comment (BenTheElder) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:65]: it was already here on line 67/65 below

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 2]
```diff
@@ -62,8 +62,6 @@ presubmits:
...
-      preset-bazel-remote-cache-enabled: "true"
       preset-dind-enabled: "true"
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:65]: @dims here ^^

Comment (dims) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:65]: whoops!

**Pull request merged**